.circleci/config.yml

```
@@ -3,7 +3,7 @@ version: 2.1
 executors:
     morpher-executor:
         docker:
-            - image: szgabsz91/jdk-ocamorph-pyphen:15.0.0
+            - image: szgabsz91/jdk-ocamorph-pyphen:latest
```

After the Docker image is released, update the reference here:

```suggestion
            - image: szgabsz91/jdk-ocamorph-pyphen:16.0.0
```